src/appkit.cs

```diff
@@ -12385,6 +12666,297 @@ interface NSProgressIndicator : NSAccessibilityProgressIndicator {
 		bool UsesThreadedAnimation { get; set; }
 	}
 
+	// Technically on NSResponder but responder subclasses can implement any that make sense
+	// So bound for user classes but not added to NSResponder binding
+	[Protocol]
```

Comment:
availability ?

Reply:
Same. Should I radar these?

Reply:
10.14 -> https://developer.apple.com/documentation/appkit/NSStandardKeyBindingResponding?language=objc